### ISSUE TITLE
OpenAPI v3: RequestBody should support OneOf, AnyOf

### ISSUE THREAD

**Issue by mhuin** [sphinx-contrib/openapi#53]
According to https://swagger.io/docs/specification/describing-request-body/ "OneOf" is a valid element to describe alternate schemas in the request body.

**Comment by ikalnytskyi** [comment]
I believe this is resolved in new renderer which uses https://github.com/sphinx-contrib/openapi/blob/2909aec06f8cf4be0ca3661e9344b5324f7c6c8b/sphinxcontrib/openapi/schema_utils.py to generate an example from schema. I'm not sure we need this to be fixed in old renderer, to be honest, because the old renderer contains enough bugs and is really hard to follow and maintain. I'd recommend everybody to move to new renderer in next major release.

**Comment by mhuin** [comment]
sweet, thanks!
